### DISCUSSION TITLE
Setting height for a window causes weird issue

### DISCUSSION THREAD

**Discussion by ryanzec** [emilk/egui#3512] [Q&A]
When I don't set a height for the window, I get this:
![2023-10-28 14_58_14-App](https://github.com/emilk/egui/assets/444206/80805e36-7dc9-4645-b04e-d9b232be91a4)

However, if I add `ui.set_height(400.0);` in the window function, I get this: 
![2023-10-28 14_57_51-App](https://github.com/emilk/egui/assets/444206/fd4575cd-0350-42e2-bd62-06a09b5d889e)

Is there a reason why my label seems to lose its margin / padding at the top when I set the width?

Full window code for context:
```rust
egui::Window::new("Tilemap Draw Tools").show(contexts.ctx_mut(), |ui| {
        ui.set_width(400.0);
        ui.set_height(400.0);
        ui.label("Tile to Draw");
        egui::ComboBox::from_id_source("Tile to Draw")
            .width(300.0)
            .selected_text(tilemap_draw_tools_state.selected_tile.as_str())
            .show_ui(ui, |ui| {
                for (tile_name, _) in tilemap_component
                    .tilemap
                    .tilemap_sprite
                    .tile_name_map
                    .iter()
                {
                    ui.selectable_value(
                        &mut tilemap_draw_tools_state.selected_tile,
                        tile_name.into(),
                        tile_name,
                    );
                }
            });
    });
```

**Comment by JumpyLionnn** [comment]
I am not sure why it happens but you can try to use the `min_height` method when building the window
https://docs.rs/egui/latest/egui/containers/struct.Window.html#method.min_height
There is one for min width as well.
